hr_attendance_modification_tracking/models/hr_attendance.py

```diff
@@ -17,3 +19,30 @@ class HrAttendance(models.Model):
     check_out = fields.Datetime(
         track_visibility='onchange'
     )
+    time_changed_manually = fields.Boolean(
+        string="Time changed",
+        required=False,
+        compute="_compute_time_changed_manually",
+        default=False,
+        store=True,
+        help="This attendance has been manually changed by user. If attendance is created from form view, a 60 "
+             "seconds tolerance will be applied."
+    )
+
+    @api.depends('message_ids.tracking_value_ids')
+    def _compute_time_changed_manually(self):
+        for record in self:
```

Comment:
Now inside of the loop you should use record instead of self, since self contains all the records and you want to process them one by one.

Reply:
Upps.
Done